### DISCUSSION TITLE
How to remove <leader>, from keymap and which key?

### DISCUSSION THREAD

**Discussion by Raagh** [LazyVim/LazyVim#2670] [Q&A] [answered]
so that is the question, I tried deleting the keymap like

![image](https://github.com/LazyVim/LazyVim/assets/8405459/073b04ec-97ed-4c5c-a553-955f64b38072)

and removing it from which-key like
![image](https://github.com/LazyVim/LazyVim/assets/8405459/29035a88-e49f-4390-9bc4-ebef7514f39f)

but it is still there

![image](https://github.com/LazyVim/LazyVim/assets/8405459/e7be23bb-11a4-440b-b77d-2db209aeec9f)

I am trying to remove quite a lot of keymaps and things from which-key as I cant stand some many options there.

**Comment by dpetka2001** [comment, marked as answer]
If you search in the repo for `<leader>,` you will get [these](https://github.com/search?q=repo%3ALazyVim%2FLazyVim%20%3Cleader%3E%2C&type=code) results and you will see that it is being defined in the `telescope.nvim` spec. And according to the [docs](https://www.lazyvim.org/configuration/plugins#%EF%B8%8F-adding--disabling-plugin-keymaps) you can disable plugin keymaps with something like 
```lua
{
  "nvim-telescope/telescope.nvim",
  keys = {
    { "<leader>,", false },
  },
}
```

**Reply by Raagh** [reply]
okay now I see it, thanks you gave me a procedure for finding and deleting this keymaps :)
